lib/emoji_data/emoji_char.rb

```diff
@@ -92,6 +103,13 @@ def variant?
       @variations.length > 0
     end
 
+    # Does the `EmojiChar` have an alternate skin variant encoding?
+    #
+    # @return [Boolean]
+    def skin_variant?
```

Comment:
That was my initial thought as well, though there's the `variant?` which also checks if there's `> 0` variants. So I decided to match the name.